src/API/Base.php

```diff
@@ -639,6 +639,13 @@ public static function validate_ads_offer_code( $advertiser_id, $offer_code ) {
 	 * @return mixed
 	 */
 	public static function get_available_discounts( $advertiser_id ) {
-		return self::make_request( "advertisers/{$advertiser_id}/discounts", 'GET', array(), 'ads' );
+		return self::make_request(
+			"advertisers/{$advertiser_id}/discounts",
+			'GET',
+			array(
+				'active' => true,
+			),
+			'ads'
+		);
```

Comment:
I think that this is supposed to be a query arg and not body parameter. Please double check but I think that this should be done like this. 
```suggestion
		$request_url = "advertisers/{$advertiser_id}/discounts";
		$request_url = add_query_arg( 'active', 'true', $request_url );

		return self::make_request( $request_url, 'GET', array(), 'ads' );
```

Reply:
Of course :man_facepalming:. Thanks for the catch